src/Resources/views/Download/creditMemo.html.twig

```diff
@@ -101,6 +101,18 @@
             </tr>
         {% endfor %}
 
+        <tr class="total">
+            <td colspan="6"></td>
+            <td>{{ 'sylius_refund.ui.net_total'|trans([], 'messages', creditMemo.localeCode) }}:</td>
+            <td>{{ '%0.2f'|format(creditMemo.getNetValueTotal()/100) }}</td>
```

Comment:
I see this logic in several places, but I'm wondering why don't we use some kind of many formatter here, just to unify extract this division to separate template